components/ApplyToHostModal.js

```diff
@@ -202,7 +202,7 @@ const ApplyToHostModal = ({ hostSlug, collective, onClose, onSuccess, router, ..
   const isOCFHost = host?.legacyId === OPENCOLLECTIVE_FOUNDATION_ID;
 
   return (
-    <Modal onClose={onClose} width="570px" {...props}>
+    <StyledModal onClose={onClose} width="570px" {...props}>
```

Comment:
Sounds like we may want to update the places using this `ApplyToHostModal`, like https://github.com/opencollective/opencollective-frontend/blob/92a45a173f8f6124a15072a85a2d697dfe7bfb91/components/accept-financial-contributions/HostCollectiveCard.js#L74

Reply:
I searched each one of these derived components, but I'm not too surprised I missed one. Thanks.